src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout/sw-in-app-purchase-checkout.spec.js

```diff
@@ -26,6 +26,9 @@ async function createWrapper(error = false) {
                         });
                     },
                     createCart: () => {
+                        if (error) {
+                            return Promise.reject(new Error('Test error'));
+                        }
```

Comment:
If not in use, please remove to avoid confusion - think it's replaced by direct function overrides, right?